scripts/metric_reporter/main.py

```diff
@@ -29,47 +32,92 @@
 logger = logging.getLogger(__name__)
 
 
-def main(config_file: str = "config.ini") -> None:
+def main(
+    config_file: str = "config.ini",
+    output_csv: bool | None = None,
+    update_bigquery: bool | None = None,
+) -> None:
     """Run the Metric Reporter.
 
     Args:
         config_file (str): Path to the configuration file.
-                           Defaults to 'ecosystem-test-scripts/config.ini'.
+        output_csv (bool | None): Whether to output CSV files. Overrides config file if provided.
+        update_bigquery (bool | None): Whether to update BigQuery tables. Overrides config file if provided.
     """
     try:
         logger.info(f"Starting Metric Reporter with configuration file: {config_file}")
         config = Config(config_file)
+        gcp_project_id: str = config.metric_reporter_config.gcp_project_id
+        bigquery_dataset_name: str = config.metric_reporter_config.bigquery_dataset_name
+        bigquery_service_account_file: str = (
+            config.metric_reporter_config.bigquery_service_account_file
+        )
+        output_csv = (
+            output_csv if output_csv is not None else config.metric_reporter_config.output_csv
+        )
+        update_bigquery = (
+            update_bigquery
+            if update_bigquery is not None
```

Comment:
I'm not sure I understand?

`update_bigquery` can be 'True', 'False' or 'None' depending on the command line input, otherwise it uses the config value, `config.metric_reporter_config.update_bigquery` if defined. The config value defaults to False if undefined.